Makefile

```diff
@@ -1,14 +1,14 @@
 .PHONY: build
 build:
-	cargo build --release
+	K8S_OPENAPI_ENABLED_VERSION=1.24 cargo build --release
 
 .PHONY: fmt
 fmt:
-	cargo fmt --all -- --check
+	K8S_OPENAPI_ENABLED_VERSION=1.24 cargo fmt --all -- --check
 
 .PHONY: lint
 lint:
-	cargo clippy --workspace -- -D warnings
+	K8S_OPENAPI_ENABLED_VERSION=1.24 cargo clippy --workspace -- -D warnings
 
 .PHONY: test
 test: fmt lint
```

Comment:
This is not a requirement, but I personally prefer setting values in variables. Thus, it's easy to change them with no code change. Something like this:

```
+API_VERSION?=1.24
+
 .PHONY: build
 build:
-       K8S_OPENAPI_ENABLED_VERSION=1.24 cargo build --release
+       K8S_OPENAPI_ENABLED_VERSION=$(API_VERSION) cargo build --release
 
 .PHONY: fmt
 fmt:
-       K8S_OPENAPI_ENABLED_VERSION=1.24 cargo fmt --all -- --check
+       K8S_OPENAPI_ENABLED_VERSION=$(API_VERSION) cargo fmt --all -- --check
 
 .PHONY: lint
 lint:
-       K8S_OPENAPI_ENABLED_VERSION=1.24 cargo clippy --workspace -- -D warnings
+       K8S_OPENAPI_ENABLED_VERSION=$(API_VERSION) cargo clippy --workspace -- -D warnings

```

Reply:
this makes a lot of sense. fixed